docs/source/user_guide/cuspatial_api_examples.ipynb

```diff
@@ -166,8 +166,9 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    cuGeoSeries = cuspatial.GeoSeries.from_points_xy(df[['longitude', 'latitude']].interleave_columns())
</pre><p>Please use PEP8 naming, e.g. <code>gs</code> or <code>cu_gs</code> or <code>cu_geo_series</code> since this is a variable, not a class.  Also <code>atl_gs</code> not <code>atlGeoSeries</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cuspatial/pull/1259/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
fixed